sentinelone/komand_sentinelone/actions/agents_shutdown/action.py

```diff
@@ -0,0 +1,25 @@
+import komand
+from komand.exceptions import ConnectionTestException
+from .schema import AgentsShutdownInput, AgentsShutdownOutput, Input, Output, Component
+
+
+class AgentsShutdown(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='agents_shutdown',
+                description=Component.DESCRIPTION,
+                input=AgentsShutdownInput(),
+                output=AgentsShutdownOutput())
+
+    def run(self, params={}):
+        agent_filter = params.get(Input.FILTER, None)
+        if "ids" not in agent_filter and "groupIds" not in agent_filter and "filterId" not in agent_filter:
+            self.logger.error("One of the following filter arguments must be supplied - ids, groupIds, filterId")
+            raise ConnectionTestException(
```

Comment:
Should be using PluginTestException as well as an assistance argument. We only use ConnectionTestException in the `test` method of `connection.py`. Fix for all